modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/in_progress_forms_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class InProgressFormsController < ApplicationController
+      def update
+        form = InProgressForm.where(form_id:, user_uuid: @current_user.uuid).first_or_initialize
```

Comment:
You should do searches by the `user_account` instead of the `user_uuid`. `user_uuid` is a historic field on in progress forms, and some day I'm going to remove it when I can get rid of all usages

Reply:
The `AccreditedRepresentativePortal::RepresentativeUser` isn't associated with a `user_account`. I can add this as a TODO